packages/math-input/src/components/keypad.tsx

```diff
@@ -140,9 +141,9 @@ class Keypad extends React.Component<Props> {
     }
 }
 
-const mapStateToProps = (state) => {
+const mapStateToProps: (state: State) => ReduxProps = (state) => {
     return {
-        ...state.echoes,
+        echoes: state.echoes.echoes,
```

Comment:
It would be nice to get rid of the extra layer of indirection.  Not necessary for this PR.